store.go

```diff
@@ -37,7 +37,8 @@ func (c *controller) closeStores() {
 	}
 }
 
-func (c *controller) getStore(scope string) datastore.DataStore {
+// GetStore returns the datastore for the given scope.
+func (c *controller) GetStore(scope string) datastore.DataStore {
```

Comment:
Why do you need to export this function ?
I can't find where it is being called from outside of libnetwork package.